charts/fybrik/files/adminconfig/default_policies.rego

```diff
@@ -4,35 +4,35 @@ package adminconfig
 
 # read capability deployment
 config[{"capability": "read", "decision": decision}] {
-    input.request.usage.read == true
+    input.request.usage == "read"
     policy := {"ID": "read-default-enabled", "description":"Read capability is requested for read workloads", "version": "0.1"}
     decision := {"policy": policy, "deploy": "True"}
 }
 
 # read capability deployment
 config[{"capability": "read", "decision": decision}] {
-    input.request.usage.read == false
+    input.request.usage != "read"
```

Comment:
The intent was "Read capability is requested only for read workloads". Anyway, I believe this and the similar policy for not writing can be removed because I can envision a flow (update) that requires both read and write.

Reply:
Removing